test/framework/src/main/java/org/elasticsearch/test/InternalTestCluster.java

```diff
@@ -440,6 +441,12 @@ private static Settings getRandomNodeSettings(long seed) {
         Random random = new Random(seed);
         Builder builder = Settings.builder();
         builder.put(TransportSettings.TRANSPORT_COMPRESS.getKey(), rarely(random));
+        builder.put(TransportSettings.TRANSPORT_COMPRESS_RAW_DATA.getKey(), random.nextBoolean());
```

Comment:
I imagine we use rarely above to avoid the overhead of compression. Can you run the full test suite with raw data compression on vs compression off to verify the penalty on builds? That would also give some indication of the overhead of lz4/raw_data.

I wonder if we only want LZ4/raw_data to run "every second run" and the other combinations (like deflate/raw_data and full compression) to only run rarely.

Reply:
I put all compression back to `rarely`. I will test this out a bit soon. Hopefully this is not a blocker as it is easy to fix if build times are impacted?

Reply:
OK, let us see how it goes, would be good to do a bit of checking after this goes in.